bioblend/_tests/TestGalaxyToolDependencies.py

```diff
@@ -0,0 +1,26 @@
+
+"""
+Test functions in bioblend.galaxy.tool_dependencies
+"""
+from . import GalaxyTestBase
+
+
+class TestGalaxyToolDependencies(GalaxyTestBase.GalaxyTestBase):
+
+    def test_summarize_toolbox(self):
```

Comment:
Judging by the test failures, I assume the API endpoint was not available in Galaxy versions < 20.01? If so, could you please add the `@test_util.skip_unless_galaxy('release_20.01')` decorator?

Reply:
Thanks!  I had not considered earlier versions